cmake/ClearBuildInfoFile.cmake

```diff
@@ -0,0 +1,5 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Create/clear file so contents can then be appended into it.
+file(WRITE "${CMAKE_BINARY_DIR}/BuildInfo.txt" "")
```

Comment:
Just online this command in the root CMakeLists.txt and delete this file? It's just one line.